session_2/cricviz/app/models/cricketer.rb

```diff
@@ -29,12 +39,20 @@ def batting_average
   #   data is incomplete
   # - If the player has not batted yet, return nil
   def batting_strike_rate
-    raise NotImplementedError
+    if innings_batted == nil || runs_scored == nil || balls_faced == 0 || balls_faced == nil
+      return nil
+    end
+    return ( runs_scored * 100 ) /  balls_faced
   end
 
   # Create records for the classical batters
   def self.import_classical_batters
-    raise NotImplementedError
+    #Name	Country	Role	Matches	Innings	Not Outs	Runs	Balls Faced	High Score	Centuries	Half-Centuries
+    Cricketer.create(name: "Sachin Tendulkar",country: "India",role: "Batter",matches: 200,innings_batted: 329,not_out: 33,runs_scored: 15921,balls_faced: 0,high_score: 248,centuries: 51,half_centuries: 68)
```

Comment:
Since the number of balls faced is missing, wouldn't writing:

```ruby
balls_faced: nil
```

be more appropriate?